packages/embedding/src/tests/attachment.test.ts

```
@@ -0,0 +1,39 @@
+import test from "ava";
+
+test("embeds attachments into provider+tenant namespace", async (t) => {
+  try {
+    const { embedAttachments } = await import("../index.js");
+    process.env.DISCORD_TOKEN_DUCK = "test";
```

ESLint

[error] Modifying an existing object/array is not allowed.

(functional/immutable-data)